sdk/cosmosdb/cosmos/test/public/integration/aggregateQuery.spec.ts

```diff
@@ -284,4 +285,66 @@ describe("Aggregate Query", function (this: Suite) {
     const response = await queryIterator.fetchAll();
     assert(response.resources.length === 0);
   });
+
+  it("should execute ORDER BY query with order on multiple fields when composite Index defined", async () => {
+    const containerDefinitionWithCompositeIndex: ContainerDefinition = {
+      id: "containerWithCompositeIndexingPolicy",
+      indexingPolicy: {
+        automatic: true,
+        indexingMode: IndexingMode.consistent,
+        includedPaths: [
+          {
+            path: "/*",
+          },
+        ],
+        excludedPaths: [
+          {
+            path: '/"systemMetadata"/*',
+          },
+        ],
+        compositeIndexes: [
+          [
+            { path: "/key", order: "ascending" },
+            { path: "/field", order: "ascending" },
+          ],
+        ],
+      },
+    };
+
+    const containerWithCompositeIndexDef = await getTestContainer(
+      "Validate multiple fields order by query",
+      undefined,
+      containerDefinitionWithCompositeIndex,
+    );
+
+    containerWithCompositeIndexDef.items.create({ id: "1", pk: "1", key: "1", field: "4" });
+    containerWithCompositeIndexDef.items.create({ id: "2", pk: "1", key: "2", field: "3" });
+    containerWithCompositeIndexDef.items.create({ id: "3", pk: "1", key: "3", field: "2" });
+    containerWithCompositeIndexDef.items.create({ id: "4", pk: "1", key: "4", field: "1" });
+    const queryIterator1 = containerWithCompositeIndexDef.items.query(
+      "SELECT * FROM r ORDER BY r.key, r.field",
+    );
+    const response = await queryIterator1.fetchAll();
+    console.log(response.resources);
+    assert(response.resources.length === 4);
+    try {
+      const queryIterator2 = containerWithCompositeIndexDef.items.query(
+        "SELECT * FROM r ORDER BY r.key DESC, r.field ASC",
```

Comment:
Why are we expecting the query to fail here? The order by is on the same `key` and `field` columns?

Reply:
we defined composite indexs in following way:
` compositeIndexes: [
          [
            { path: "/key", order: "ascending" },
            { path: "/field", order: "ascending" },
          ],
        ],`
 observe order of both fields as ascending, but in query:
 `"SELECT * FROM r ORDER BY r.key DESC, r.field ASC",`
ordering of one field is descending and for other it's ascending, which would is invalid as per:
https://learn.microsoft.com/en-us/azure/cosmos-db/index-policy#composite-indexes